resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -62,17 +63,26 @@ export default {
       showWizardTemplates: false,
       showCardProcesses: false,
       category: null,
+      numCategories: 15,
+      page: 1,
     };
   },
   mounted() {
     this.getCategories();
   },
   methods: {
+    addCategories() {
+      this.page += 1;
+      this.getCategories();
+    },
     getCategories() {
       ProcessMaker.apiClient
-        .get("process_categories")
+        .get(`process_categories?page=${this.page}&per_page=${this.numCategories}`)
         .then((response) => {
-          this.listCategories = response.data.data;
+          const newCategories = response.data.data;
```

Comment:
Use the spread operator, like this
`this.listCategories = [...this.listCategories, ...response.data.data]`